docker/php/Dockerfile

```diff
@@ -1,33 +1,24 @@
-FROM php:7.3.5-apache
+FROM ubuntu:18.04
 MAINTAINER Dastan Rahimi <dastan.rahimi@ablevets.com>
 
-RUN apt-get update && apt-get install -y wget
+ENV DEBIAN_FRONTEND noninteractive
+RUN apt-get update -q && apt-get install software-properties-common -y && \
+	add-apt-repository ppa:ondrej/php && apt-get update && \
+	apt-get install -y python python-pip \
+	git composer nginx php7.3-fpm php7.3-intl php7.3-common php7.3-mbstring \
+	php7.3-zip php7.3-cli php7.3-gd php7.3-zip php7.3-simplexml php7.3-xml \
+	php7.3-pdo php7.3-curl php7.3-imagick pkg-config \
+	php7.3-mysql gcc make autoconf libc-dev pkg-config php7.3-dev
+
+RUN apt-get update && apt-get install -y libpng-dev zlib1g-dev git subversion zlib1g-dev libzip-dev net-tools iputils-ping zip mysql-client vim wget nginx ssmtp
```

Comment:
Can we run imagejpeg() on with these libs? I originally added: libjpeg62-turbo-dev for the function to work